pom.xml

```diff
@@ -641,7 +648,7 @@
              <showWarnings>true</showWarnings>
              <compilerArgs>
                <compilerArg>-Werror</compilerArg>
-               <compilerArg>-Xlint:deprecation</compilerArg>
+               <!--compilerArg>-Xlint:deprecation</compilerArg-->
```

Comment:
Why comment it out?
Please just remove if not needed. But I think it is intentional to fail the build if we use deprecated code.

Reply:
Done